libraries/rush-lib/src/logic/base/BaseInstallManager.ts

```diff
@@ -450,11 +451,37 @@ export abstract class BaseInstallManager {
         // Clear the currently installed git hooks and install fresh copies
         FileSystem.ensureEmptyFolder(hookDestination);
 
+        // Find the relative path from Git hooks directory to the directory storing the actual scripts.
+        const hookRelativePath: string = Path.convertToSlashes(path.relative(hookDestination, hookSource));
+
         // Only copy files that look like Git hook names
         const filteredHookFilenames: string[] = hookFilenames.filter((x) => /^[a-z\-]+/.test(x));
         for (const filename of filteredHookFilenames) {
-          // Copy the file.  Important: For Bash scripts, the EOL must not be CRLF.
-          const hookFileContent: string = FileSystem.readFile(path.join(hookSource, filename));
+          // Make sure the actual script in the hookSource directory has required permission bits
+          const hookFilePath: string = path.join(hookSource, filename);
```

Comment:
I have no problem doing that, however, why is that better? Is it a convention in this repo? I usually assume using `path.join` can prevent possible issues with FS on various OS.

Reply:
We've been trying to standardize on using `/` as the directory separator everywhere (the fs APIs all accept input in this format) because it simplifies path operations and makes them faster. It is still necessary to use `path.join` or `path.resolve` if one of the paths may contain `.` or `..` elements, though.

Reply:
Updated